pcdsdevices/sxr_test_absorber.py

```diff
@@ -0,0 +1,22 @@
+"""
+Module for the SXR Test Absorbers.
+"""
+from ophyd.device import Component as Cpt
+from ophyd.device import Device
+
+from .epics_motor import BeckhoffAxis
+
+
+class SxrTestAbsorber(Device):
+    """
+    SXR Test Absorber: Used for testing the sxr beamline at high pulse rates.
+
+    This device has 1 main members: the stopper/absorber (diamond stopper).
+
+    The vertical motor moves stopper in and out of the beam path in
+    the +/- y direction.
+    """
+
+    tab_component_names = True
+
+    absorber_vert = Cpt(BeckhoffAxis, ':absorber_vert', kind='normal')
```

Comment:
Does this class have anything that isn't just a motor? If not, t may be better to just use `BeckhoffAxis` directly rather than make a special class for it.

Reply:
As of right now no. There may be more additions in the future. There are Thermocouples that may be added in the future. The hardware is there, but the specifications were very lax for installation.

Reply:
Also, there was no mps requirement for this device but we should still have it because it can block beam.

Reply:
I can see both sides of this (keep the class, or just use `BeckhoffAxis`), but some thoughts in favor of keeping it:
* It's a place for documentation
* It's a place where further signals can be added, if needed
* Its special class name allows for a custom screen to be easily created by filename (though certainly in the one motor case, the axis in happi could have this screen information)

Reply:
If there are other things to add later then :+1: